### ISSUE TITLE
PSA: FSR 3.1 decouples upscaler and frame generator.

### ISSUE THREAD

**Issue by TimmyP777** [Nukem9/dlssg-to-fsr3#404]
Making this mod no longer necessary. Thank you Nukem9!

Here is your answer to how 2000/3000 series owners get official frame generation support: AMD.

**Comment by MarcoReckless** [comment]
> Making this mod no longer necessary. Thank you Nukem9!
> 
> Here is your answer to how 2000/3000 series owners get official frame generation support: AMD.

Correct me if I'm wrong, but this only means we will be able to use FSR3 Frame Gen with DLSS; on the other side, this mod replaces DLSS Frame Gen with FSR3 Frame Gen when FSR3 is not implemented in the first place (like in Cyberpunk 2077, at least at the moment).

**Comment by TimmyP777** [comment]
Yeah but every game from here on out will include both, but I guess youre right.

**Comment by LeBlackMeow** [comment]
The first game to support FSR 3.1 will be Ratchet & Clank, and only, later this year. Other games will follow-up and probably next year.

For now, this mod is still necessary.

**Comment by VicentAAV** [comment]
It would be good if they updated the mod with FSR 3.1 because surely there will be several games that do not implement FSR 3 or 3.1

**Comment by jesstech** [comment]
> It would be good if they updated the mod with FSR 3.1 because surely there will be several games that do not implement FSR 3 or 3.1

FSR 3.1 is a spatial upscaler and has no relation on the AMD Fluid Motion Frames temporal upscaler that this mod uses to replace DLSS Frame Generation.

**Comment by xCaptainTroy** [comment]
This mod not only necessary for combining the DLSS wil FG its also important to solve the bad FG implementation

**Comment by TimmyP777** [comment]
> > It would be good if they updated the mod with FSR 3.1 because surely there will be several games that do not implement FSR 3 or 3.1
> 
> FSR 3.1 is a spatial upscaler and has no relation to the AMD Fluid Motion Frames temporal upscaler that this mod uses to replace DLSS Frame Generation.

Fluid motion frames has absolutely nothing to do with this, whatsoever. Its not even using the same hardware.

FSR3 is doing about *exactly* the same thing as DLSS3. On RTX 2000/3000 it even passes data to the Optical Flow Accelerator just like DLSS3-FG, this is why it works so well on the 3000 series... the cards that I suspect actually have an updated NVOFA chip.

**Comment by jesstech** [comment]
Ah, I misunderstood how it works under the hood then.  I had assumed it redirected  upscaling calls in DLSS games to AMD’s FSR3 dll, but after trying it on a 3080 and noticing no visible fizzling like FSR is known for, I concluded it was in fact only redirecting the motion interpolation portion to FSR3’s FMF flow in order to maintain DLSS spatial upscaling.
